### PR TITLE
Prepare for deployment on npm

### PR DESCRIPTION
**Pull request by Nudelsieb**
We might want to consider adding automated testing with Chrome Headless or something as a hook before deploy to make sure the transpiled package works in Chrome. We could use the same tests we use for unit testing.

I also need some guidance regarding versioning. I am not sure if v1.0.0 is right or if we need to add the complete DSL spec first.

### CODE REVIEW

**Comment by myabc** [comment]
There's something on the Babel page about targeting different browsers.

A bit more background on `package.json` fields here: https://github.com/defunctzombie/package-browser-field-spec

As a general rule, it's better to transpile as little as possible. However for the time-being if want to deliver different code to different environments (browsers/ES-module/CJS-supporting node versions and tool), this means adopting a build/bundling tool.  This could be Webpack, but [Rollup](https://github.com/rollup) is simpler and better suited to library code.

Example here: https://github.com/OUP/mymaths-content-format/blob/master/packages/mymaths-xml-validation/rollup.config.ts

File: package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "rando",
-  "version": "0.0.0",
+  "version": "1.0.0",
   "description": "Simple random number generator",
   "main": "src/rando.js",
   "scripts": {
+    "build": "babel src -d dist",
```
Comment (myabc) [inline package.json]: @Nudelsieb I've never used it, but we might want to play with Buble some time:

http://buble.surge.sh/guide/#using-the-javascript-api
https://github.com/KwanMan/jest-buble

Comment (Nudelsieb) [reply package.json]: Looks interesting, but need to read more on this.

**Comment by Nudelsieb** [comment]
What about this in the package.json:
```
"browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 8"
  ]
```

File: package.json (hunk 1 above)
Comment (Nudelsieb) [reply package.json]: I think the Vue CLI installs buble by default.

**Pull request closed without merging**